src/apollo-client/__tests__/exports.js

```diff
@@ -0,0 +1,27 @@
+/** Copyright (c) 2018 Uber Technologies, Inc.
```

Comment:
dat year though 😮😅
(I'm kidding, but I'm sure this is going to be fun to fix in all of our files)